WickedEngine/wiLua.h

```diff
@@ -3,9 +3,15 @@
 
 extern "C"
 {
+#ifdef _WIN32
```

Comment:
It's better to just use forward slash everywhere

Reply:
Indeed. I'm not sure why Windows even uses \\ when it supports /.